annotations/views/rest_views.py

```diff
@@ -492,59 +493,23 @@ def search(self, request, **kwargs):
         if not q:
             return Response({'results': []})
         pos = request.GET.get('pos', None)
-        url = f"{settings.CONCEPTPOWER_ENDPOINT}ConceptLookup/{q}/{pos if pos else ''}"
-        response = requests.get(url, auth=(settings.CONCEPTPOWER_USERID, settings.CONCEPTPOWER_PASSWORD))
+        url = f"{settings.CONCEPTPOWER_ENDPOINT}ConceptSearch"
+        parameters = {
+            'word': q,
+            'pos': pos if pos else '',
```

Comment:
So if `''` is passed for `pos` then this will result in a 400 error from ConceptPower. Passing `None` will actually return 'Any' pos